doc/ref_cert/lfn/chapters/chapter04.md

```diff
@@ -82,8 +82,28 @@ It covers the test cases against identity management operations like user manage
 | `cert.test.vim.01` | API | Show API endpoint catalog. It *must* have endpoint for each Core service. | `req.int.api.01` |
 
 #### 4.3.4.2 Image - Glance
-It covers the test cases against image management operations.
 
+Glance API is covered in the OpenStack Gates via
+[Tempest](https://opendev.org/openstack/tempest) as integrated in
+[Functest Smoke CNTT](https://git.opnfv.org/functest/tree/docker/smoke-cntt/testcases.yaml).
+
+According to
+[RA1 Core OpenStack Services APIs](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md)
```

Comment:
Use relative links. This will redirect wrongly when compiled with jekyll on `https://cntt-n.github.io/`